guides/common/modules/con_managing-alternate-content-sources.adoc

```diff
@@ -2,9 +2,24 @@
 = Managing Alternate Content Sources
 
 Alternate content sources define alternate paths to download content during synchronization.
-The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+The content itself is downloaded from the alternate content source and only the metadata is downloaded from the {ProjectServer}.
```

Comment:
Metadata can come from the "upstream URL" or Foreman depending on the situation.  If the ACS is created on the Foreman's main smart proxy, then the metadata will come from some upstream server.  If the ACS is created on a remote smart proxy, then the metadata will come from the Foreman.

I suppose you could just say something like "upstream server or Foreman server depending on the configuration".